spec/models/user_spec.rb

```
@@ -0,0 +1,658 @@
+        
+        expect(json['parent']['id']).to eq(parent_user.id)
+        expect(json['parent']['name']).to eq(parent_user.name)
+      end
+    end
+
+    describe '#generate_jwt' do
+      let(:user) { create(:user, role: student_role, institution: institution) }
+
+      it 'generates a valid JWT token' do
+        token = user.generate_jwt
+        
```

RuboCop (1.86.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)